src/main/java/net/tiagofar78/prisonescape/bukkit/BukkitWorldEditor.java

```diff
@@ -131,6 +135,37 @@ public static void clearDirtFromMazePart(Location upperCorner, Location lowerCor
 
     public static void fillWithBars(Location upperCorner, Location lowerCorner) {
         fill(upperCorner, lowerCorner, Material.IRON_BARS);
+        connectBars(upperCorner, lowerCorner);
+    }
+
+    private static void connectBars(Location upperCorner, Location lowerCorner) {
+        for (int x = lowerCorner.getBlockX(); x <= upperCorner.getBlockX(); x++) {
+            for (int y = lowerCorner.getBlockY(); y <= upperCorner.getBlockY(); y++) {
+                for (int z = lowerCorner.getBlockZ(); z <= upperCorner.getBlockZ(); z++) {
+                    Location currentLocation = new Location(lowerCorner.getWorld(), x, y, z);
+                    BlockData bd = Bukkit.createBlockData("minecraft:iron_bars");
+
+                    if (bd instanceof MultipleFacing) {
```

Comment:
I don't think you need this if. Iron bars are being created immediately before this call.